string task_name
ItemTarget[] target_items
